NineChronicles.Headless.Executable/appsettings.json

```diff
@@ -84,5 +88,38 @@
                 }
             }
         ]
+    },
+    "Headless": {
+        "AppProtocolVersionString": "100291/6ec8E598962F1f475504F82fD5bF3410eAE58B9B/MEUCIQCA3lzUAt0QBfG.+ezw4CQ69zBy669sANEt5juSJgzqcgIgbozfpcyeuKJDeJoT5exyGYDYBqCpxklsMEfs0SQ6qzo=/ZHUxNjpXaW5kb3dzQmluYXJ5VXJsdTcyOmh0dHBzOi8vcmVsZWFzZS5uaW5lLWNocm9uaWNsZXMuY29tL21haW4vdjEwMDI5MS9sYXVuY2hlci92MS9XaW5kb3dzLnppcHU5OnRpbWVzdGFtcHUxMDoyMDIyLTA5LTA3ZQ==",
+        "GenesisBlockPath": "https://release.nine-chronicles.com/genesis-block-9c-main",
+        "StoreType": "rocksdb",
+        "StorePath": "~/AppData/Local/planetarium/9c-main-partition",
```

Comment:
Something I'm slightly worried about is that `~` is a POSIX convention and may not work on .NET or Windows.

Reply:
It's little bit tricky. In this case I prefer to put placeholder like `[path/to/9c-main-partition]` but this `appsetting.json` is not an example so placeholder is not good choice to pick.
But there is no silver bullet to cover everything...